tools/buildsys/src/gomod.rs

```diff
@@ -0,0 +1,151 @@
+/*!
+Packages using the Go programming language may have upstream tar archives that
+include only the source code of the project, but not the source code of any
+dependencies. The Go programming language promotes the use of "modules" for
+dependencies and projects adopting modules will provide go.mod and go.sum
```

Comment:
nit:
```suggestion
dependencies and projects adopting modules will provide `go.mod` and `go.sum`
```

(same nit below)